docs/HowToUseAudio.md

```diff
@@ -20,7 +20,7 @@ the lower half through POSIX like system APIs.
 
 
 ##  Tinyalsa  
-[framework/src/tinyalsa](../framework/src/tinyalsa) has the Tiny library similar to ALSA in Linux. Refer [link to TINYALSA !](https://github.com/tinyalsa/tinyalsa)  
+[framework/src/tinyalsa](../framework/src/tinyalsa) has the Tiny library to provide audio API similar to ALSA in Linux. Refer [HowToUseTinyAlsa](HowToUseTinyalsa.md) to find out how to develop audio applications using TinyAlsa
```

Comment:
Please add some comments that tinyalsa using some functionalites in audio subsystem.

Reply:
Also, instead of 'Refer [HowToUseTinyAlsa](HowToUseTinyalsa.md) to find out how to develop audio applications using TinyAlsa'
'To use tinyalsa in application, refer [How to use Tinyalsa in User Guide](HowToUseTinyalsa.md) blah blah blah.